src/main/resources/swagger.api/schemas/error.json

```diff
@@ -0,0 +1,26 @@
+{
+  "description": "An error",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "message": {
+      "type": "string",
+      "description": "Error message text"
+    },
+    "type": {
+      "type": "string",
+      "description": "Error message type"
+    },
+    "code": {
+      "type": "string",
+      "description": "Error message code"
+    },
+    "parameters": {
+      "type": "object",
+      "description": "Error message parameters",
+      "$ref": "parameters.json"
+    }
+  },
+  "additionalProperties": false,
+  "required": [ "message" ]
```

Comment:
Is only `message` required, not `code` and/or `type`? Also earlier `nptError.sample` has only `type` and `code`, but no `message`.

Reply:
Yes, for NPE errors java does not provide by default error message, but this is rare case, that should not be occurred in production. And I suppose this validation only works in request body, but not in response